packages/react-charts/src/components/Chart/Chart.tsx

```diff
@@ -283,7 +285,11 @@ export interface ChartProps extends VictoryChartProps {
    * Note: When adding a legend, padding may need to be adjusted in order to accommodate the extra legend. In some
    * cases, the legend may not be visible until enough padding is applied.
    */
-  legendPosition?: 'bottom' | 'bottom-left' | 'right';
+  legendPosition?: 'bottom' | 'bottom-left' | 'bottom-right' | 'left' | 'right';
```

Comment:
How does the legend position change when supporting RTL text. Trying to understand why the legend must be relocated with 'bottom-right' and 'left'?

Don't see 'left' used anywhere.

Reply:
The left and bottom-right positions would be to support RTL alignment of the chart as a whole. The `legendDirection` prop is for RTL text in the legend.

Maybe someone from design could offer input about whether chart legends should be on the left for RTL?